src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -2698,4 +2698,76 @@ limitations under the License.
             <include cluster="Device Energy Management Mode" client="false" server="true" clientLocked="true" serverLocked="false"></include>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-heatpump</name>
+        <domain>CHIP</domain>
+        <typeName>Heat Pump</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0309</deviceId>
+        <class>Simple</class>
+        <scope>Endpoint</scope>
+        <clusters>
+            <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true">
+                <requireAttribute>CLIENT_LIST</requireAttribute>
```

Comment:
Why are all these requireAttribute for mandatory attributes here?  looks like just noise...

Reply:
Similar comment was raised on #36125 - it looks like this is something some clusters have and Alchemy has included.

Reply:
@bzbarsky-apple @hasty is there a decision on what to do here? Are we okay to leave these in or should I manually strip them out.

I notice that #36125 has merged already which presumably has these left in?